src/lib/components/forms-header.svelte

```diff
@@ -0,0 +1,9 @@
+<script>
+  export let label = '';
+</script>
+
+<div class="flex w-[99vw] flex-row place-self-center border-b-[3px] border-b-muted-red-500 p-[2vw]">
+  <p class="ml-11 text-5xl font-bold">
+    {label}
+  </p>
```

Comment:
```suggestion
<div class="grow border-b-[3px] border-b-muted-red-500 pr-[2vw] pb-[2vw] pt-[1vw] mx-5">
  <p class="ml-5 text-5xl font-bold">
    {label}
  </p>
</div>
```

This suggestion is based on some storybook tests where in mobile the bar extends a bit too much, it's better to do it this way which tells the component to greedily take up as much space as possible!